website/api/controllers/webhooks/receive-from-github.js

```diff
@@ -199,24 +199,36 @@ module.exports = {
         let isAutoApproved = await sails.helpers.flow.build(async()=>{
 
           let isSenderDRIForAllChangedPaths = false;
+          let isSenderMaintainer = GITHUB_USERNAMES_OF_BOTS_AND_MAINTAINERS.includes(sender.login);
           let DRI_BY_PATH = {
             'README.md': 'mike-j-thomas',// (github brandfront)
-            'handbook': 'mikermcneil',// (default for handbook)
-            'handbook/README.md': 'mikermcneil',
+
+            'handbook': ['desmi-dizney', 'mike-j-thomas', 'mikermcneil'],// (default for handbook)
```

Comment:
@Desmi-Dizney fyi, this will auto-approve your PRs to the handbook so that you can do small grammatical edits for sentence case, punctuation, etc in the existing handbook pages, and merge them yourself without waiting.

(that said, if the change contains anything more than pure grammar, would you please request review from the page maintainer so they can read your changes before merging?)

Reply:
This is helpful. I'll do so.